newton/_version.py

```
@@ -1 +1,6 @@
-__version__ = "1.1.0.dev0"
+try:
+    from importlib.metadata import version
+
+    __version__ = version("newton")
+except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)